lib/journaled.rb

```diff
@@ -4,9 +4,11 @@
 require "request_store"
 
 require "journaled/engine"
+require "journaled/job_priority"
 
 module Journaled
   mattr_accessor :default_app_name
+  mattr_accessor(:default_job_priority) { Journaled::JobPriority::EVENTUAL }
```

Comment:
TIL! I didn't know you could pass a block as a default like this, but I really like it!

Apparently on Rails 5.2+ you can just pass a `default` option to `mattr_accessor` but we only specify 5.1 here